lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs

```diff
@@ -156,6 +156,16 @@ ProtocolParameters
     Foreign Wallet fk_wallet_protocol_parameters protocolParametersWalletId ! ON DELETE CASCADE
     deriving Show Generic
 
+-- Track whether the wallet's stake key is registered or not.
+StakeKeyCertificate
+    stakeKeyCertWalletId             W.WalletId     sql=wallet_id
+    stakeKeyCertSlot                 W.SlotId       sql=slot
+    stakeKeyCertIsReg                Bool           sql=is_reg
```

Comment:
I'd recommend using something more explicit than a boolean here. Boolean convey very little information and they are easy to misinterpret (like I did on my first review...). I'd suggest something like `Registration | Deregistration`, stored as plain text lowercased in the database, so it's also readable when debugging / inspecting.